src/detect-file-hash-common.c

```diff
@@ -345,6 +345,10 @@ int DetectFileHashSetup (DetectEngineCtx *de_ctx, Signature *s, const char *str,
     if (type == DETECT_FILESHA256) {
         s->file_flags |= FILE_SIG_NEED_SHA256;
     }
+
+    /* Matching on a hash require filestore. */
+    s->flags |= SIG_FLAG_FILESTORE;
```

Comment:
I'm pretty sure this isn't correct. Is there a SV test already?

Reply:
Based on the data provided in the issue:
https://github.com/OISF/suricata-verify/pull/132